contentcuration/contentcuration/management/commands/setup.py

```diff
@@ -260,6 +261,7 @@ def create_file(display_name, preset_id, ext, user=None):
             original_filename=display_name,
             preset_id=preset_id,
             uploaded_by=user,
+            language_id="en" if FormatPreset.objects.filter(id=preset_id, multi_language=True).exists() else None,
```

Comment:
@jayoshih  What is this about?

Would the language code `mul` be a better fit for this? 
https://github.com/learningequality/le-utils/blob/master/le_utils/resources/languagelookup.json#L6 (added in recenlty in le-utils 0.10)

Reply:
Sounds good!